python/sklearn.Dockerfile

```diff
@@ -39,9 +39,14 @@ ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 
 RUN useradd kserve -m -u 1000 -d /home/kserve
 
-COPY --from=builder --chown=kserve:kserve $VIRTUAL_ENV $VIRTUAL_ENV
-COPY --from=builder kserve kserve
-COPY --from=builder sklearnserver sklearnserver
+# Make all files also owned by the root group
```

Comment:
do we also have to add the `kserve` user to group `0` above?

```Dockerfile
RUN useradd kserve -m -u 1000 -g 0 -d /home/kserve
```

Reply:
I don't think so. The group association is not relevant when running under the user "kserve" since, for this, all the user-permission flags are the relevant ones (as before).

For OpenShift, when the process is _not_ running as "kserve" but with an arbitrary UID that belongs to the root group, the group-permission flags are relevant. 

The Dockerfiles set the group-permission flags to the same values as the user-permission flags (that `chmod -R g=u`), so both cases (running under "kserve" and running under any user belonging to group 0) will be equivalent.